### DISCUSSION TITLE
Authentication type : UserAssignedManagedIdentity or SystemAssignedManagedIdentity Example

### DISCUSSION THREAD

**Discussion by alexandergoncharovaspecta** [tomkerkhove/promitor#2336] [Q&A]
Hello 

I can not find real example of using Authentication type UserAssignedManagedIdentity. 
I mean the next questions:
1. Example terraform of creating managed identity - UserAssignedManagedIdentity with accesing for example to sql resource group
2. Example values.yaml for case Authentication type  UserAssignedManagedIdentity

Have a nice day

**Comment by tomkerkhove** [comment]
I'd recommend looking at terraform if you need any samples.

For managed identity you can configure it through the helm chart https://artifacthub.io/packages/helm/promitor/promitor-agent-scraper

**Reply by alexandergoncharovaspecta** [reply]
Thank for your time. I use promitor-agent-scraper for a long time but for authentication, I used ServicePrincipal. Now I have a task to redo for using managed identity, and sure as Devops I use terraform for infrastructure and I have created managed identity with role MonitorReader. My main problem that I did not find any example for promitor-agent-scrapper for case UserAssignedManagedIdentity and steps for kubernetes to use the one. I know that there is default managed identity for k8s and I use the one for access to keyvault but the task is : create the new managed identity with role MonitorReader and use it for promitor.

**Reply by tomkerkhove** [reply]
Unfortunately I can only help with configuring Promitor so I'd advise to get the managed identity work done and circle back once the configuration with Promitor is failing
